### PR TITLE
GROW-240 banner

### PR DESCRIPTION
**Pull request by ryan-ludwig**
Bump storybook version
Build and animate Swashie
Starting point of KvProgressCircle
Allow theming for KvProgressBar
Mostly duplicate AppealBanner component as AppealBanner15 and add to 15year story.

### CODE REVIEW

File: src/pages/15Years/15Years.vue [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
-					<fifteen-years-sponsors />
-				</section>
-				<section>
-					<fifteen-years-how-kiva-works />
-				</section>
-			</fifteen-years-styles>
-		</div>
-	</www-page>
+	<div>
+		<appeal-banner-15 :appeal-banner-content="{}" />
```
Comment (ryan-ludwig) [inline src/pages/15Years/15Years.vue:3]: I don't really understand how appeal-banner works, but i'm pretty sure this isn't where it should go.

Comment (mcstover) [reply src/pages/15Years/15Years.vue:3]: Correct. The global promo banner and any type of appeal is selected in the GlobalPromotionalBannerContentful.vue file. Selection is based on the first found Banner from the contentful setting AND the type of the Banner.

File: src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue [hunk 2]
```diff
@@ -0,0 +1,562 @@
...
+	</div>
+</template>
+
+<script>
+import _forEach from 'lodash/forEach';
+import _get from 'lodash/get';
+import numeral from 'numeral';
+import store2 from 'store2';
+
+import appealBannerQuery from '@/graphql/query/appealBanner.graphql';
```
Comment (mcstover) [inline src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue:105]: The query in the global promo banner will get the setting and banner from contentful. We'll need to create a new query for this file that uses the new graphql query we've yet to build.

File: src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue [hunk 3]
```diff
@@ -0,0 +1,562 @@
+<template>
```
Comment (mcstover) [inline src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner15.vue:1]: Lots of cleanup can be done in this file.

File: src/pages/15Years/15Years.vue (hunk 1 above)
Comment (mcstover) [reply src/pages/15Years/15Years.vue:3]: So you can just remove it here as in will be included with www-page

**Review by mcstover — APPROVED** [review]
Thanks for pushing it forward some more!

**Pull request merged**